ci/pipeline.yml

```diff
@@ -126,6 +128,84 @@ jobs:
         status: failure
         context: unit-tests
 
+  - name: shipit
+    public: true
+    serial: true
+    plan:
+    - do:
+      - in_parallel:
+          - { get: git,     passed: [rc] }
+      - task: release
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: cf-routing.common.repositories.cloud.sap/pcap-release-testflight
+              tag:        semantic
+              username:   ((docker.username))
+              password:   ((docker.password))
+          inputs:
+            - name: git
+          outputs:
+            - name: gh
+          run:
+            path: ./git/ci/scripts/shipit
+            args: []
+          params:
+            RELEASE_NAME: pcap-release
+            REPO_ROOT:    git
+            RELEASE_ROOT: gh
+            BRANCH:        main
+            GITHUB_OWNER:  cloudfoundry
+            GITHUB_TOKEN: ((github.access_token))
+            GCP_SERVICE_KEY: ((gcp.service_key))
+
+      - put: blobstore
+        params:
+          file:  "gh/artifacts/pcap-*.tgz"
+      on_failure:
+        put: notify
+        params:
+          channel:  "#pcap-release"
+          username: ci-bot
+          icon_url: "((slack.icon))"
+          text:    "((slack.fail_url)) pcap-boshrelease : shipit job failed"
+
+  - name: rc
+    public: true
+    serial: true
+    plan:
+    - do:
+      - in_parallel:
+          - { get: git,     passed: [unit-tests] }
+      - task: release-candidate
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: cf-routing.common.repositories.cloud.sap/pcap-release-testflight
+              tag:        semantic
+              username:   ((docker.username))
+              password:   ((docker.password))
+          inputs:
+            - name: git
+          outputs:
+            - name: gh
+          run:
+            path: ./git/ci/scripts/shipit
+            args: []
+          params:
+            RELEASE_NAME: pcap-release
+            REPO_ROOT:    git
+            RELEASE_ROOT: gh
+            BRANCH:        main
+            DRY_RUN: true
+            GITHUB_OWNER:  cloudfoundry
+            GITHUB_TOKEN: ((github.access_token))
+            GCP_SERVICE_KEY: ((gcp.service_key))
```

Comment:
The indentation / alignment is a bit all over the place in this file, particularly here. Maybe we can align or bite the bullet and IDE auto format?

Reply:
let's decide on a formatter and use it everywhere. takes tomato vs tomato out of the picture.